File: src/client/lib/api/models/Fingerprint.dart
import 'package:sc_utility/api/models/AssetFile.dart';

class Fingerprint {
  String sha;
  String version;
  List<AssetFile> files;

  Fingerprint.fromJson(Map<String, dynamic> json) {
    sha = json["sha"];
    version = json["version"];
    files = (json["files"] as List).map((p) => AssetFile.fromJson(p)).toList();
  }
}

File: src/api/Models/Fingerprint/Fingerprint.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace SupercellUilityApi.Models
{
    public class Fingerprint
    {
        [JsonPropertyName("files")] public List<AssetFile> Files { get; set; }
        [JsonPropertyName("sha")] public string Sha { get; set; }
        [JsonPropertyName("version")] public string Version { get; set; }


    }
}
